x-pack/plugins/beats_management/public/components/layouts/no_data.tsx

```diff
@@ -4,13 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem, EuiPageContent } from '@elastic/eui';
 import React from 'react';
 import { withRouter } from 'react-router-dom';
 
-import { EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem, EuiPageContent } from '@elastic/eui';
-
 interface LayoutProps {
-  title: string;
+  title: string | JSX.Element;
```

Comment:
I've personally had some weird issues with `JSX.Element` and usually just reach for `React.ReactNode` at this point, but I know there are some differences there that you may not want.